Source/UI/SecurityCodeComponent/SecurityCodeComponent.swift

```diff
@@ -9,16 +9,26 @@ import Checkout
 import UIKit
 
 public final class SecurityCodeComponent: UIView {
-  private var view: SecurityCodeView!
+  // Wrapped and protected inner view
+  var view: SecurityCodeView!
+  
+  // func configure(_:_:) arguments
+  var configuration: SecurityCodeComponentConfiguration!
+  var isSecurityCodeValid: ((Bool) -> Void)!
 
-  private var configuration: SecurityCodeComponentConfiguration!
-  private var isSecurityCodeValid: ((Bool) -> Void)!
-  private var cardValidator: CardValidating!
+  // func configure(_:_:) initialised properties. They depend on the configuration argument
```

Comment:
What is the purpose of this comment?

Reply:
Juts an internal one. Categorising the properties. Otherwise I tend to leave a blank line in between the different functional groupings of properties and it doesn't always look good without a proper explanation @deepesh-vasthimal-cko so, no harm in adding these.

Reply:
resolving this but feel free to re-open for further discussion